hw03_altyyevaa.py

```diff
@@ -0,0 +1,224 @@
+# Author: Aziza Altyyeva
+# Username: altyyevaa
+#
+# Assignment: to create a creative project using functions, turtles, loops and etc.
+# Purpose: to practice what I have learned until today
+# Google Doc Link : https://docs.google.com/document/d/1qbbG98mVSgm59Zz0Qic8VSSfr2YC18v-gFslQo-V63c/edit
+
+
+import turtle
+
+def drawHouse1(lomi):  #draws the house
+    lomi.pensize(10)
+    turtle.colormode(255)
+    lomi.pencolor((63, 37, 8))
+    turtle.colormode(255)
+    lomi.penup()
+    lomi.goto(-300, -300)
+    lomi.pendown()
+    lomi.fillcolor("#3F2508")
+    lomi.begin_fill()
+    for i in range(4):  #draws the square of the house
+        lomi.forward(200)
+        lomi.left(90)
+    lomi.end_fill()
+    lomi.penup()
+    lomi.left(45)
+    lomi.forward(290)
+    lomi.down()
+    lomi.fillcolor("#D5AB7B")
+    lomi.begin_fill()
+    for a in range(2): #draws the roof the house
+        lomi.left(90)
+        lomi.forward(150)
+    lomi.end_fill()
+    lomi.penup()
+    lomi.backward(150)
+    lomi.right(135)
+    lomi.pendown()
+    lomi.pencolor('black') #draws the flag
+    lomi.forward(70)
+    lomi.color("#19AC3B")
+    lomi.fillcolor("#19AC3B")
+    lomi.begin_fill()
+    for b in range (3): #draws the square shape of the flag
+        lomi.right(90)
+        lomi.forward(40)
+    lomi.end_fill()
+def drawRoad(lomi): #draws a gray straight road
+    lomi.pensize(10)
+    turtle.colormode(255)
+    lomi.pencolor("#B6B6B2")
+    lomi.penup()
+    lomi.right(240)
+    lomi.forward(400)
+    lomi.pendown()
+    lomi.fillcolor("#B6B6B2")
+    lomi.begin_fill()
+    lomi.left(135)
+    lomi.forward(645)
+    lomi.right(75)
+    lomi.forward(90)
+    lomi.right(70)
+    lomi.forward(665)
+    lomi.left(-110)
+    lomi.forward(487)
+    lomi.end_fill()
+    lomi.penup()
+    lomi.pencolor("black")
+    lomi.pensize(20)
+    lomi.right(160)
+    lomi.forward(245)
+    lomi.pendown()
+def drawLines (lomi): #draws the black middle lines to make it actually appear like a road
+        lomi.setheading(0)
+        lomi.left(90)
+        lomi.pensize(40)
+        lomi.forward(120)
+        lomi.penup()
+        lomi.forward(90)
+        lomi.pendown()
+
+        lomi.setheading(0)  #line no2
+        lomi.pensize(30)
+        lomi.left(90)
+        lomi.forward(120)
+        lomi.penup()
+        lomi.forward(90)
+        lomi.pendown()
+
+        lomi.setheading(0)  #line no3
+        lomi.pensize(25)
+        lomi.left(90)
+        lomi.forward(105)
```

Comment:
The indentation on this function is different than the indentation elsewhere in the code. They should all be aligned the same.

Reply:
Did you make any changes to the code? Because some parts have been kinda changed (I mean my picture has changed slightly) also this part of code is indented just like other functions. Did you change it or what am I missing?